packages/grafana-runtime/src/utils/queryResponse.ts

```diff
@@ -13,15 +13,24 @@ import {
   FieldType,
   DataQuery,
 } from '@grafana/data';
+import { FetchResponse } from '../services';
 
-interface DataResponse {
+export interface DataResponse {
   error?: string;
   refId?: string;
+  // base64 encoded arrow tables
   dataframes?: string[];
   series?: TimeSeries[];
   tables?: TableData[];
 }
 
+/**
```

Comment:
Added some typings and exported some of them here as they help with testing @ryantxu @kylebrandt 
@marefr

Reply:
can we add the @private annotation?  Ideally this is not used externally and can match the response object

Reply:
Seems like private is not valid tag here:
`Unsupported block tag: @private`
https://drone.grafana.net/grafana/grafana/15000/1/20